recipe/meta.yaml

```
@@ -1,88 +1,108 @@
-    - libopus # [not win]
-    - openssl # [not win]
-  run:
-    - lame  # [not win]
+    - aom         3.6.0
+    - bzip2       {{ bzip2 }}
+    - dav1d       1.2.1
+    - freetype    {{ freetype }}
+    - fontconfig  {{ fontconfig }}
+    - harfbuzz    {{ harfbuzz }}
+    - libiconv    1.16
+    - zlib        {{ zlib }}
+    - openh264    2.1.1           # [not win]
+    - openjpeg    {{ openjpeg }}  # [not win]
+    - librsvg     2.54.4          # [osx]
+    - libtheora   1.1.1
+    - libvorbis   1.3.7           # [not win]
+    - libxml2     {{ libxml2 }}
+    - tesseract   5.2.0           # [not (win or s390x)]
+    - lame        3.100           # [not win]
+    - libvpx      1.13.1          # [not (win or s390x)]
+    - libopus     1.3             # [not win]
+    - openssl     {{ openssl }}
+    - xz          5.4.6
+    - libglib     2.78.4          # [osx]
+    - cairo       1.16.0          # [osx]
 
 test:
   commands:
     - ffmpeg --help
-    - ffmpeg -loglevel panic -codecs
-    - ffmpeg -loglevel panic -protocols | grep "https"  # [not win]
-    - ffmpeg -loglevel panic -codecs | grep "libmp3lame"  # [not win]
-    - ffmpeg -loglevel panic -codecs | grep "DEVI.S zlib"  # [unix]
-    - ffmpeg -loglevel panic -codecs | grep "DEV.LS h264"  # [linux64 or osx64]
-    - ffmpeg -loglevel panic -codecs | grep "D.V.LS h264"  # [ppc64le]
-    - ffmpeg -loglevel panic -codecs | grep "libx264"  # [linux64 or osx64]
-    - ffmpeg -loglevel panic -codecs | grep "libopenh264"  # [linux64 or osx64]
+    - ffmpeg -loglevel panic -protocols | grep "https"     # [not win]
+    - ffmpeg -loglevel panic -codecs | grep "libmp3lame"   # [not win]
+    - ffmpeg -loglevel panic -codecs | grep "DEVI.S zlib"  # [not win]
+    - ffmpeg -loglevel panic -codecs | grep "DEV.LS h264"  # [not (win or s390x)]
+    - ffmpeg -loglevel panic -codecs | grep "libopenh264"  # [not win]
     # Verify dynamic libraries on all systems
     {% set ffmpeg_libs = [
         "avcodec",
         "avdevice",
         "swresample",
-        "postproc",
         "avfilter",
-        "swresample",
         "avcodec",
         "avformat",
-        "swscale",
-        "avresample"
+        "swscale"
     ] %}
     {% for each_ffmpeg_lib in ffmpeg_libs %}
-    - test -f $PREFIX/lib/lib{{ each_ffmpeg_lib }}.dylib  # [osx]
-    - test -f $PREFIX/lib/lib{{ each_ffmpeg_lib }}.so     # [linux]
+    - test -f $CONDA_PREFIX/lib/lib{{ each_ffmpeg_lib }}.dylib  # [osx]
+    - test -f $CONDA_PREFIX/lib/lib{{ each_ffmpeg_lib }}.so     # [linux]
+    - if not exist %PREFIX%\Library\bin\{{ each_ffmpeg_lib }}*.dll exit 1  # [win]
+    - if not exist %PREFIX%\Library\bin\{{ each_ffmpeg_lib }}.lib exit 1  # [win]
     {% endfor %}
 
 about:
-  home: http://www.ffmpeg.org/
-  license: GPL 3
-  license_file: COPYING.GPLv3  # [unix]
+  home: https://www.ffmpeg.org
+  license: GPL-2.0-or-later AND GPL-3.0-or-later AND LGPL-3.0-or-later
+  license_file:
+    - COPYING.LGPLv3
+  license_family: OTHER
```

```suggestion
  license_family: LGPL
```
or `GPL`. THough, that field is deprecated and not used, so it doesn't really amtter.